content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Visualizing Your Mesh
+description: This task shows you how to visualize your services within an Istio mesh.
+weight: 50
+keywords: [telemetry,visualization]
+---
+
+This task shows you how to visualize different aspects of your Istio mesh.
+
+As part of this task, you will install the [Kiali](https://www.kiali.io) add-on and use
+the web-based graphical user interface for viewing service graphs of the mesh
+as well as your Istio configuration objects such as virtual services
+and destination rules. You will also use the Kiali API to generate graph data
+in the form of consumable JSON.
+
+The [Bookinfo](/docs/examples/bookinfo/) sample application is used as
+the example application throughout this task.
+
+## Before you begin
+
+{{< idea_icon >}} If you already have Istio installed and need Kiali installed, you can follow the [Kiali install instructions here](https://www.kiali.io/gettingstarted/).
```

Comment:
this looks ok, but the recommended way to achieve what you want is to use `helm template --set kiali.enabled --set (other kiali flags needed)` and kubectl apply it.  There is a similar workflow for helm upgrade which does the same thing.  What is proposed here would result in an inconsistent deployment. I'd recommend keeping the link around, but simply as a reference, not as the cononical way to install Kiali on Istio.

Reply:
@sdake OK, I moved that down to the bottom of the "before you begin" section just as a reference in case someone wants an alternative to helm.

I wasn't sure if we should assume the reader had Istio already installed or not - I tried not to assume that, but rather try to keep it relevant whether they have Istio installed or not (i.e. the instructions have them run helm template and then kubectl apply - which, as you say, can install Istio, too, or just update it with a kiali install).